src/web-server.js

```diff
@@ -332,6 +332,75 @@ app.delete("/services/:serviceName", function (req, res) {
     });
 });
 
+app.delete("/scopes", function (req, res) {
+  let body;
+  if (typeof req.body === "string") {
```

Comment:
In what situation would the request body not be a string? Is this if statement for when there is an empty request body? As far as I can tell, request bodies are always parsed as a string in this project, due to the line of code below.

https://github.com/single-spa/import-map-deployer/blob/b6c0ca778137d67587c77eff8dcef57d1c6021f7/src/web-server.js#L38